tests/Struct/FindologicProductTest.php

```diff
@@ -459,9 +459,29 @@ private function getKeywordEntity(string $keyword): ProductSearchKeywordEntity
         return $productSearchKeywordEntity;
     }
 
-    public function testProduct(): void
+    public function hasManufacturerProvider(): array
     {
-        $productEntity = $this->createTestProduct();
+        return [
+            'Product with manufacturer' => [
+                'hasManufacturer' => true,
```

Comment:
This is preference, but I typically avoid boolean values for data providers, as with every boolean you have to check the code to grasp what's going on. In this case it is simple, but once we add more things here, it may get hard to read. In this example I would simply set `manufacturerName` to either `FINDOLOGIC` or `null`. It is fine the way it is now, unless you already want to change that, but please just keep this in mind for the future.

Reply:
I think this would make more sense, if you could set the manufacturer name `null` and set it in `$overrideData`.
Renaming it, to be consistent with the parameter of `createTestProduct`.

Reply:
See https://github.com/findologic/plugin-shopware-6/pull/254#discussion_r822446617.